.github/workflows/test-linting.yml

```diff
@@ -8,12 +8,12 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18.x]
+        node-version: ['20']
```

Comment:
Same here, we can remove the matrix configuration and configure the `actions/setup-node` action directly:

```yml
    steps:
      - uses: actions/checkout@v4
      - name: Use Node.js
        uses: actions/setup-node@v4
        with:
          node-version: 20.x
```

This prevents the action name from changing when we update the node version 🙂

Reply:
@langemike I will update required checks when you move node-version definition to the `actions/setup-node@v4`.

Reply:
I made changes in the required checks.